modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -385,6 +386,14 @@ static bool _on_core_api_assembly_loaded() {
 void GDMono::initialize() {
 	print_verbose(".NET: Initializing module...");
 
+#ifdef TOOLS_ENABLED
+	// Exit early if we're going to be generating glue code, so we can bootstrap the bindings without needing to download a separate mono-enabled godot binary
+	List<String> cmdline_args = OS::get_singleton()->get_cmdline_args();
+	if (BindingsGenerator::is_generating_glue(cmdline_args)) {
+		print_verbose(".NET: GodotPlugins initialized");
```

Comment:
```suggestion
		print_verbose(".NET: Skipping runtime initialization because of glue generation.");
```

Or some other not misleading message.

Reply:
This is fixed in #76659